cms/templates/cms/wizards/base.html

```diff
@@ -15,8 +15,5 @@
 
 {% block extrahead %}
     {{ block.super }}
-    <script type="text/javascript" src="{% static_with_version "cms/js/modules/jquery.noconflict.pre.js" %}"></script>
```

Comment:
I'm not sure if we can remove it here, I remember that dependencies such as select2 required the namespaced django jQuery?